trusted-committer-script.md

```diff
@@ -0,0 +1,55 @@
+# InnerSource Learning Path Script: Trusted Committer
+
+## Introduction of speakers
+
+- D: tbd
+
+- G: My name is Georg Grütter. I am a software developer and an InnerSource
+  Evangelist at Robert Bosch in Germany and I was a Trusted Committer in one of
+  our InnerSource communities for many years.
+
+## Introducing The Trusted Committer Role
+
+D: The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill. It goes
+far beyond being an opinionated gatekeeper and it is instrumental for the 
+success of any InnerSource community.  
+
+G: Generally speaking, the TC role is defined by its responsibilities, rather
+than by its privileges. On a very high level, a TC represents the interests of
+both their InnerSource community and the products the community is building.
+She is concerned with the health of the community and the product. So as a TC,
+you'll have both tech oriented and community oriented responsibilities and
+we'll explore both of these dimensions in the following sections. 
+
+D: Before we go into the details of what a TC actually does, let's spend some
+time contrasting the TC role to other roles in IS and why we think the name is
+both apt and important.
+
+G: There are three major roles in an InnerSource community: The contributor,
+the trusted committer and, depending on the type of community, a product owner.
+
+D: The Contributor, as the name suggests, makes contributions.  She might or
+might not be part of the community - sometimes they come from another part of
+the organization to develop a feature for it.  This is why in InnerSource, we
+sometimes refer to contributors as _guests_. It is the responsibility of the
+contributor to "fit in" and conform to the communities expectations and
+processes - to honor the house rules, so to speak.
+
+G: The Trusted Committers, on the other hand, are, among other things, 
+responsible for setting and enacting these rules. This is why TCs are sometimes
+referred to as hosts. They are responsible to make sure their guests are
+comfortable and can make the most of their time spent in the community. In
+comparison to the contributor, a TC has more rights. She can push code closer
+to production and she is generally allowed to perform tasks that have a higher
+level of risk associated with them. A TC is at the center of an InnerSource 
+project and thus collaborates with both the contributors and the Product Owner.
+
+D: The Product Owner (PO) is responsible for defining and prioritizing
+requirements and stories for the community to implement. Together with the PO,
+she will e. g. determine, if a feature donated by a contributor actually
+belongs to the product she is responsible for. Depending on the type of 
+InnerSource community, there might or might not be a Product Owner.
```

Comment:
I think there _always_ will be a Product Owner role being filled in a successful inner source project.  The distinction is that it may not always be a separate person from the TC - one person can fill both roles.

Reply:
You are right, @rrrutledge . I had the roles of both PO and TC at the same time.